### PR TITLE
Changes for ROS Noetic support

### PR DESCRIPTION
**Pull request by MichaelGrupp**
Adds a Dockerfile for ROS Noetic and fixes the package.xml

### CODE REVIEW

File: cartographer_ros/CMakeLists.txt [hunk 1]
```diff
@@ -48,6 +48,11 @@ find_package(cartographer REQUIRED)
 include("${CARTOGRAPHER_CMAKE_DIR}/functions.cmake")
 option(BUILD_GRPC "build features that require Cartographer gRPC support" false)
 google_initialize_cartographer_project()
+# PCL requires C++14 since version 1.10.0 - override if needed.
+if (NOT ${PCL_VERSION} VERSION_LESS "1.10.0")
```
Comment (MichaelGrupp) [inline cartographer_ros/CMakeLists.txt:52]: `if (${PCL_VERSION} VERSION_GREATER_EQUAL "1.10.0")` would be nicer, but this doesn't work with CMake 3.5 from Ubuntu 16.

File: .travis.yml [hunk 2]
```diff
@@ -24,6 +24,7 @@ cache:
 env:
   - ROS_RELEASE=kinetic DOCKER_CACHE_FILE=/home/travis/docker/kinetic-cache.tar.gz
   - ROS_RELEASE=melodic DOCKER_CACHE_FILE=/home/travis/docker/melodic-cache.tar.gz
+  - ROS_RELEASE=noetic DOCKER_CACHE_FILE=/home/travis/docker/noetic-cache.tar.gz
```
Comment (MichaelGrupp) [inline .travis.yml:27]: Noetic CI requires changes in cartographer (python3-sphinx etc), this should fix it: https://github.com/cartographer-project/cartographer/pull/1705

File: cartographer_ros/CMakeLists.txt [hunk 3]
```diff
@@ -48,6 +48,11 @@ find_package(cartographer REQUIRED)
 include("${CARTOGRAPHER_CMAKE_DIR}/functions.cmake")
 option(BUILD_GRPC "build features that require Cartographer gRPC support" false)
 google_initialize_cartographer_project()
+# PCL requires C++14 since version 1.10.0 - override if needed.
```
Comment (MichaelGrupp) [inline cartographer_ros/CMakeLists.txt:51]: Would it make sense to not explicitly set C++11 in `google_initialize_cartographer_project()`?

Comment (wohe) [reply cartographer_ros/CMakeLists.txt:51]: cartographer-project/cartographer#1724

Comment (MichaelGrupp) [reply cartographer_ros/CMakeLists.txt:51]: Thanks, I will change it after that's merged.

**Comment by MichaelGrupp** [comment]
There is still an error during the libcartographer build, as I mentioned here: https://github.com/cartographer-project/cartographer/pull/1705/files#r452321342
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
/catkin_ws/src/cartographer/GMOCK_INCLUDE_DIRS
```
https://travis-ci.org/github/cartographer-project/cartographer_ros/jobs/711095782

**Comment by MichaelGrupp** [comment]
The build has passed now succesfully.

**Pull request merged**